tests/Ecommerce/CartTaxManagementTest.php

```diff
@@ -59,7 +59,7 @@ private function buildTaxClass(array $taxes = [], $combinationType = TaxEntry::C
         return $taxClass;
     }
 
-    private function setUpProduct($grossPrice, array $taxes = [], string $combinationType = TaxEntry::CALCULATION_MODE_COMBINE): CheckoutableInterface
+    private function setUpProduct(int $grossPrice, array $taxes = [], string $combinationType = TaxEntry::CALCULATION_MODE_COMBINE): CheckoutableInterface
```

Comment:
```suggestion
    private function setUpProduct(float|int|string|Decimal $grossPrice, array $taxes = [], string $combinationType = TaxEntry::CALCULATION_MODE_COMBINE): CheckoutableInterface
```
Maybe we should use more than int here like in the DecimalTest?

Reply:
Its a private method and its only used with integers. But we can extend it, should not harm.

Reply:
Thank you! If we ever decide to test it with something different than an int, we just need to change the input, instead of also changing the types.